style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,230 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Naming elements
+
+### `aria-label` and `aria-labelledby`
+
+Every element on a page will have a name, that is read out to an assistive technology
+like a screen reader. This will for most elements be the content of the element.
+For form elements it will be the content of the assosciated `<label>` (see below).
+
+You can overwrite that name, that is read out, by specifying a new name via the
+`aria-label` attribute. This must e.g. be done, if the element itself has no
+visual text representation (e.g. an icon button):
+
+```html
+<button aria-label="Add filter"><i class="fa fa-plus"></i></button>
```

Comment:
Font awesome will work on any inline tag, and possibly any block-level tag (haven't tested).